integration-tests/tests/test_utils/trait_mod.rs

```diff
@@ -0,0 +1,347 @@
+use axelar_wasm_std::nonempty;
+use axelar_wasm_std::MajorityThreshold;
+use connection_router_api::ChainName;
+use cosmwasm_std::{Addr, Binary, Deps, Env, StdResult, Uint256};
+use cw_multi_test::{App, ContractWrapper, Executor};
+use integration_tests::contract::Contract;
+use multisig::key::KeyType;
+use multisig_prover::encoding::Encoder;
+use service_registry::contract::{execute, instantiate, query};
+
+#[derive(Clone)]
+pub struct ServiceRegistryContract {
+    pub contract_addr: Addr,
+}
+
+impl ServiceRegistryContract {
```

Comment:
Do we want to actually add the execution msgs and query msgs here as methods? So someone can just call something like `contract.get_active_workers(service_name, chain_name)` as a function?